aws/fargate/Dockerfile

```diff
@@ -14,5 +15,7 @@ COPY UI/ /src/main/webapp
 COPY src/main/webapp/ /src/main/webapp
 COPY node_modules/ /src/main/webapp/node_modules
 COPY target/scala-2.11/digiroad2-assembly-0.1.0-SNAPSHOT.jar /digiroad2.jar
+COPY aws/fargate/startup.sh /startup.sh
+RUN sed -i 's/\r$//' /startup.sh  && chmod +x /startup.sh
```

Comment:
nämä pystyy asettamaan gitissä oikein, ettei ole tarvetta tehdä näin. Ei se toki mitään haitttaakaan, mutta voisi olla paikallaan laittaa tuo scripti gitissä kuntoon. Eli linux rivinvaihdot ja ajo-oikeudet.